### PR TITLE
KHP3-6174 - (feat) added radiology  price display on the basket

### PR DESCRIPTION
**Pull request by its-kios09**
## Requirements
- [ ] This PR has a title that briefly describes the work done including a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) type prefix and a Jira ticket number if applicable. See existing PR titles for inspiration.

#### For changes to apps
- [ ]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).

#### If applicable
- [ ] My work includes tests or is validated by existing tests.

## Summary

-  added ability to show radiology prices on the order basket

https://thepalladiumgroup.atlassian.net/browse/KHP3-6174

## Screenshots

radio

### CODE REVIEW

File: src/form/radiology-orders/add-radiology-order/radiology-order-form.component.tsx [hunk 1]
```diff
@@ -205,6 +206,11 @@ export function RadiologyOrderForm({
         id="radiologyOrderForm"
       >
         <div className={styles.form}>
+          <ExtensionSlot
+            name="top-of-lab-order-form-slot"
```
Comment (donaldkibet) [inline src/form/radiology-orders/add-radiology-order/radiology-order-form.component.tsx:210]:
```suggestion
            name="top-of-imaging-order-form-slot"
```

We might also want to change this repo to `esm-imaging-app`

**Review by donaldkibet — CHANGES_REQUESTED** [review]

File: src/form/radiology-orders/add-radiology-order/radiology-order-form.component.tsx (hunk 1 above)
Comment (its-kios09) [reply src/form/radiology-orders/add-radiology-order/radiology-order-form.component.tsx:210]: so, what do you advise?

Comment (ojwanganto) [reply src/form/radiology-orders/add-radiology-order/radiology-order-form.component.tsx:210]: renaming the app to imaging

Comment (its-kios09) [reply src/form/radiology-orders/add-radiology-order/radiology-order-form.component.tsx:210]: Okay, how about creating the slot from esm-billing-app

Comment (its-kios09) [reply src/form/radiology-orders/add-radiology-order/radiology-order-form.component.tsx]: > We might also want to change this repo to `esm-imaging-app`

Sorted

**Comment by ojwanganto** [comment]
Tagging @makombe. Thanks, @its-kios09

**Review by makombe — APPROVED** [review]
Thanks @its-kios09. The ask to rename esm to imaging is still pending. I think that one you can address in a followup PR.

**Pull request merged**